### DISCUSSION TITLE
Discuss Mods We Want to Test

### DISCUSSION THREAD

**Discussion by delizin** [delizin/traitormod#14] [Ideas]
Discuss the mods that we might want to test for compatibility. 

Here are two that are confirmed working and recommended for everyone to add client side as well:
- Client-side Performance Fix: https://steamcommunity.com/sharedfiles/filedetails/?id=2701251094
- Lua for Barotrauma https://steamcommunity.com/sharedfiles/filedetails/?id=2559634234

**Comment by CrazySwayze** [comment]
Ok, here's a mod wishlist sorted from the ones I care about most to least:
- Dynamic Europa: https://steamcommunity.com/sharedfiles/filedetails/?id=2532991202
   - Bigger map and better persistence for a long game (outposts & new enemy dens grow & conflict with each other)
   - More challenging and varied biomes, enemies, and missions
   - Food consumption and eating enemies. Perfect for us goons.
   - Currently there's an issue with the "waste infection" in this mod interacting with Barotraumatic (my next recommended mod). Based on their explanation it seems relatively easy to resolve but is worth mentioning.

- Barotraumatic: https://steamcommunity.com/sharedfiles/filedetails/?id=2518816103
   - Number 1 mod for more varied and challenging monsters. Need I say more?
   - Same author as Dynamic Europa, they're compatible but need to be loaded in a certain order
   - Also replaces the default radio with 5 new ones of varying reliability. Could be interesting.
   - Also requires:
      - Barotraumatic Creature Pack: https://steamcommunity.com/workshop/filedetails/?id=2831987252
      - Improved Husks: https://steamcommunity.com/workshop/filedetails/?id=2085783214

- Let Them All In: https://steamcommunity.com/sharedfiles/filedetails/?id=2805065898
   - LET THEM ALL IN

- ADV Beacon Stations: https://steamcommunity.com/sharedfiles/filedetails/?id=2544952900
   - Like 45 and they're all high quality, easy win here
   - Nice accompaniment for Dynamic Europa since I think that touches everything EXCEPT Beacon Stations.

- Movable and Sellable Wrecks: https://steamcommunity.com/sharedfiles/filedetails/?id=2772926011
   - Sick. Dock wrecks to your boat, haul them to a station, and sell them.
   - Highly recommended with this mod: New Wrecks for Barotrauma (With sellable wrecks): https://steamcommunity.com/sharedfiles/filedetails/?id=2184257427
      - Adds 60+ new wrecks with traps and shit. Mega cool.

- Barocraftables+: https://steamcommunity.com/sharedfiles/filedetails/?id=2764140582
   - With a bigger ship comes more personalized areas and it would be great to be able to furnish them a lil
   - Also adds stuff like the ID card printer which could be hilarious when used with something like Jobs Extended
   - Author recommends using this with LUA Linker for obvious reasons
   - Some possible funky interactions with bots when repairing repairable items
   - Overrides some vanilla items to make them printable, might need looking at & compatibility check. Included in the Barotraumatic mod pack so I assume it's at least compatible with the stuff in there.

- More Artifacts (ACOS): https://steamcommunity.com/sharedfiles/filedetails/?id=2776026770
   - Looks really cool, adds non-mission artifacts that behave more like ss13 artifacts
   - Not 100% sure if this is working or not, some people in the comments were having issues but the dev was still replying and fixing stuff. Think it might be ok or just need some minor cleanup.

- Combine Ammo Boxes: https://steamcommunity.com/sharedfiles/filedetails/?id=2835362298
   - Nice quality of life, requires clientside mode too

- Better Health UI: https://steamcommunity.com/sharedfiles/filedetails/?id=2807158602
   - It's really nice. The regular health UI sucks.
   - Requires Cs For Barotrauma for some reason: https://steamcommunity.com/workshop/filedetails/?id=2795927223

- Server Host Helper: https://steamcommunity.com/sharedfiles/filedetails/?id=2850108374
   - Creates rollback points every time you change maps
   - Dynamic addition and removal of bots! Can configure it to fill up to a certain server size with bots.
   - No clue if this works with 1.0

- Storage Icons: https://steamcommunity.com/sharedfiles/filedetails/?id=2920200449
   - Adds an icon to secure containers showing the most numerous item in them. Seems rad.

- Disconnect Kill and Stun Remover: https://steamcommunity.com/sharedfiles/filedetails/?id=2658872348
   - People stay around until the map changes instead of flopping over dead, kinda minor but might be appreciated
   - Server side

- Jobs Extended: https://steamcommunity.com/sharedfiles/filedetails/?id=2526236489
   - Might be a bit much for now, though it could add a lot to traitor mode
   - A bit more personalization so that people can feel like they own their roles
   - Fun interactions with ID Printer from Barocraftibles+, plus people could set up their own little nooks
   - Might counterintuitively save us time. For example, standardizes all clothes to have pockets.
   - Can use either a collection of compatible subs (I assume different areas of the sub are ID locked) or "semi-auto-convert" a sub using code in the mod description page.

- PocketsForAll (Draconis) Forked: https://steamcommunity.com/sharedfiles/filedetails/?id=2900856197
   - A bunch of sensible clothing changes. Seems pretty nice honestly.
   - Alternative or companion to Jobs Extended depending on load order

- Complete Global Darkness (foced autorun for clients): https://steamcommunity.com/sharedfiles/filedetails/?id=2883996067
   - Fuck you Rejd
   - Seem fun and spooky
   - Prevents spectator shenanagins
   - Removes ability to highlight buttons and other objects but you can turn it back on by deleting "NoItemHighlight.lua"

- Alarms Extended: https://steamcommunity.com/sharedfiles/filedetails/?id=2638494991

- Zenith's Sunken Tapes (No Afflictions): https://steamcommunity.com/sharedfiles/filedetails/?id=2918966067
   - Sunken tapes, boomboxes, and walkmen

- Laser Attachment Pointer: https://steamcommunity.com/sharedfiles/filedetails/?id=2899922407
   - Cool simple useful. Goes in flashlight slot.

- Sparky's Gunnery: https://steamcommunity.com/sharedfiles/filedetails/?id=2895083880
   - Adds 6 fun new guns, actually seems like the guy earnestly attempted to balance them too
   - Slightly buggy but probably nothing serious. Adds 2 new toxin cabinets, wierdly, and some ammo box storage crates.

- Crates for Recycling: https://steamcommunity.com/sharedfiles/filedetails/?id=2811526690
   - In Russian but seems like it adds 2 crates that have 1 slot for each resource, so you pop em both in to the recycler. Nerds might appreciate this one, idk.
